Formula/luaradio.rb

```diff
@@ -0,0 +1,47 @@
+class Luaradio < Formula
+  desc "lightweight, embeddable flow graph signal processing framework for SDR"
+  homepage "http://luaradio.io/"
+  url "https://github.com/vsergeev/luaradio/archive/v0.1.0.tar.gz"
+  sha256 "4423f94c0bb259952eae7db11e4db4509dfa7c6e205798c35a095050c530212b"
+
+  depends_on "pkg-config" => :build
+  depends_on "luajit"
+  depends_on "fftw" => :recommended
+
+  def install
+    cd "embed" do
+      system "make", "install", "PREFIX=#{prefix}"
+    end
+  end
+
+  test do
+    # Create text file which will be read and processed by LuaRadio
```

Comment:
> The way Homebrew PRs work is we request you make changes and you either make our changes as requested or close the PR.

Where's that written? I did not find that in the contribution guideline or the code of conduct.

I included that comment because this was the first time I used LuaRadio (which might result in poor code quality) and processing text files is rather unusual for a software defined radio library/tool but I felt it might be a good simple test. The comments where placed to make that somewhat clear and therefore an extra file is needed.

Honestly I feel a bit upset that I have to justify myself for my opinion on such a trivial issue and I'm again disappointed that I'm getting encountered in such a rough way as to me it seems like your opinion the only one that matters.

I like to enable new tools to be available through Homebrew but this nitpicking on trivial, non-blocking issues bothers me almost every time in a way that I want to quit doing that.